src/routes/Register/Register.module.css

```
@@ -1,15 +1,132 @@
-.submitBtn {
-  background-color: skyblue;
-  border: 2px solid lightgray;
+.registerInput {
+  width: 90%;
+  height: 3.5rem;
   display: inline-block;
-  margin-top: 1rem;
-  padding: 1rem;
-  font-size: 1.3rem;
+  background-color: #fcfcfc;
+  border: 1px solid #eeeeee;
+  border-radius: 4px;
+  padding: 0 0 0 1rem;
+  border-bottom: 1px solid #eeeeee;
 }
-.textBox{
-  padding: 0.5rem 14rem 0.5rem 0.5rem;
-  display: block;
+.inputBorderActive {
+  border-bottom: 2px solid #3058c0;
 }
-.textBoxWrap{
-  margin-top: 3rem;
+.inputTextUnder {
+  color: red !important;
+  font-size: 11px !important;
+  position: relative;
+  top: 0.7rem;
+  left: 0.3rem;
+}
+.inputWrap {
+  padding: 1rem 0 1rem 2rem;
+  margin-bottom: 0.5rem;
+}
+.inputWrap > ul > li {
+  width: 4.5rem;
+  display: inline-block;
+  margin-right: 1rem;
+}
+.inputWrap > ul > li > input {
+  display: none;
+}
```

```suggestion
.inputWrap {
  padding: 1rem 0 1rem 2rem;
  margin-bottom: 0.5rem;
  
  > ul > li {
    width: 4.5rem;
    display: inline-block;
    margin-right: 1rem;
    
    > input {
      display: none;
    }
  }
```

SCSS에서는 CSS 처럼 쓸 필요가 없습니다.